apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -560,17 +580,21 @@ def _get_board_from_makefile(self, makefile_path: pathlib.Path) -> str:
     FLASH_MAX_RETRIES = 5
 
     def flash(self, options):
+        import serial.tools.list_ports
```

Comment:
Add a comment explaining why we can't have this at the top with the rest of the imports?

Reply:
done. We should fix this issue. It's only because we are running microtvm tutorials in GPU where the dependencies are not installed